Whoaverse/Whoaverse/Scripts/whoaverse.js

```diff
@@ -204,7 +204,119 @@ $(document).ready(function () {
             }
         });
     });
-
+    
+    var key = {};
+
+    // List of the inputs the user use when creating a new discussion
+    var submitInputs = {
+      post: [
+        '#Title',
+        '#Subverse',
+        '#MessageContent'
+      ],
+      reply: [
+        '#CommentContent'
+      ]
+    };
+    
+    document.onkeydown = function(evt) {
```

Comment:
Why do you use a global listener instead of one attached to the element? Also jQuery is included so you could use that.